tests/unittests/TensorsTest.cpp

```diff
@@ -1300,9 +1302,12 @@ TEST(Tensor, differentAlignment) {
 TEST(Tensor, accessToRawTextFile) {
   Tensor tensorRef = {0.75f,  0.23f, 0.76f,  0.99f,  1.00f,
                       -0.78f, 0.23f, -0.97f, -0.37f, 0.00f};
-  dumpToRawTextFile(tensorRef, "tensor.txt");
+  llvm::SmallString<64> path;
+  auto tempFileRes = llvm::sys::fs::createTemporaryFile("tensor", ".txt", path);
```

Comment:
@jfix71 It seems like your PR resulted in OSS CI failures.
```
sccache /usr/bin/clang++-8  -DFMT_LOCALE -DGIT_DATE=\"2020-03-16\" -DGIT_SHA1=\"fc0567c\" -DGLOW_WITH_CPU=1 -DGLOW_WITH_LLVMIRCODEGEN=1 -DWITH_PNG -I../include -Iinclude -I. -I../ -I../externalbackends -I../thirdparty/fp16/include -I../tests/unittests -I../thirdparty/miniz-2.0.8 -I../thirdparty/folly -Ithirdparty/folly -isystem /usr/lib/llvm-8/include -isystem ../tests/googletest/googletest/include -isystem ../tests/googletest/googletest -Werror -Wall -Wnon-virtual-dtor -g -fno-omit-frame-pointer -O0 -fPIE -fvisibility=hidden -fvisibility-inlines-hidden   -pthread -std=c++14 -MD -MT tests/unittests/CMakeFiles/TensorsTest.dir/TensorsTest.cpp.o -MF tests/unittests/CMakeFiles/TensorsTest.dir/TensorsTest.cpp.o.d -o tests/unittests/CMakeFiles/TensorsTest.dir/TensorsTest.cpp.o -c ../tests/unittests/TensorsTest.cpp
../tests/unittests/TensorsTest.cpp:1306:8: error: unused variable 'tempFileRes' [-Werror,-Wunused-variable]
  auto tempFileRes = llvm::sys::fs::createTemporaryFile("tensor", ".txt", path);
```

See e.g. https://circleci.com/gh/pytorch/glow/62670?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link for more details.

Could you fix it? Thanks!